schemas/iso19139/src/main/plugin/iso19139/index-fields/index-subtemplate.xsl

```diff
@@ -142,19 +138,20 @@
   </xsl:template>
 
 
-
+  <!-- Indexing distribution formats -->
   <xsl:template mode="index" match="gmd:MD_Format[count(ancestor::node()) =  1]">
     <xsl:variable name="title"
                   select="if (gmd:version/gco:CharacterString = '' or gmd:version/gco:CharacterString = '-')
                         then gmd:name/gco:CharacterString
                         else concat(gmd:name/gco:CharacterString, ' ', gmd:version/gco:CharacterString)"/>
     <resourceTitle><xsl:value-of select="$title"/></resourceTitle>
```

Comment:
@fxprunayre should be the same for distribution formats?

```suggestion
    <resourceTitleObject type="object">{
      "default": "<xsl:value-of select="gn-fn-index:json-escape($title)"/>"
      }</resourceTitleObject>
```

Reply:
Yes would be nice. ISO19115-3 is fine for this https://github.com/geonetwork/core-geonetwork/blob/main/schemas/iso19139/src/main/plugin/iso19139/index-fields/index-subtemplate.xsl. So we only need to update this here.